app/main.py

```diff
@@ -1,8 +1,24 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = []
+    for data in people:
+        name = data["name"]
+        age = data["age"]
+        wife = data.get("wife")
+
+        person = Person(name, age)
+        person_list.append(person)
+
+        if wife and wife in Person.people:
+            person.wife = Person.people[wife]
+            Person.people[wife].husband = person
```

Comment:
The attributes `wife` and `husband` are added dynamically to `Person` instances. This can lead to potential issues if these attributes are accessed before being set. Consider defining these attributes in the `Person` class with a default value of `None` to avoid such issues.